src/NHibernate/Cache/UpdateTimestampsCache.cs

```diff
@@ -26,6 +26,8 @@ public virtual void Clear()
 			updateTimestamps.Clear();
 		}
 
+		// Since v5.1
+		[Obsolete("Please use extension overload with an ICache parameter.")]
```

Comment:
This is a constructor, not an extension.

Reply:
After your class interface, my turn to get caught by "too quick copy/paste"!